sal_interfaces/MTAOS/MTAOS_Commands.xml

```diff
@@ -54,14 +54,21 @@
   <SALCommand>
     <Subsystem>MTAOS</Subsystem>
     <EFDB_Topic>MTAOS_command_preProcess</EFDB_Topic>
-    <Description>Pre-process image for WEP.</Description>
+    <Description>Pre-process image for WEP, e.g., ISR.</Description>
```

Comment:
suggest: "Pre-process image for WEP. The outputs of this command are donut images that are ready for curvature wavefront sensing".